tests/experimental/pydantic/test_basic.py

```diff
@@ -63,6 +64,48 @@ class UserType:
     assert field2.type.of_type is str
 
 
+def test_basic_type_with_input_metadata():
+    class User1(pydantic.BaseModel):
+        friend: Optional[str] = "friend_value"
+        frenemy: str
+
```

Comment:
can we add another testcase / statement with
```
friend: Optional[str] = None 
```

default value with None is tricky in pydantic context...

Reply:
Done, 4070d5f.
Let me know if another change is desired.